src/PowerShell.Core.Instrumentation/PowerShell.Core.Instrumentation.man

```diff
@@ -2199,6 +2205,7 @@
               name="PowerShellCore/Operational"
               symbol="C_OPERATIONAL"
               type="Operational"
+              access="O:BAG:SYD:(A;;0xf0007;;;SY)(A;;0x7;;;BA)(A;;0x7;;;SO)(A;;0x1;;;S-1-5-32-573)"
               >
```

Comment:
The inbox has these additional permissions:

```
InputObject                                                                       SideIndicator
-----------                                                                       -------------
NT AUTHORITY\BATCH: AccessAllowed (ListDirectory, WriteData)                      <=
NT AUTHORITY\INTERACTIVE: AccessAllowed (ListDirectory, WriteData)                <=
NT AUTHORITY\SERVICE: AccessAllowed (ListDirectory, WriteData)                    <=
NT AUTHORITY\WRITE RESTRICTED: AccessAllowed (ListDirectory, WriteData)           <=
APPLICATION PACKAGE AUTHORITY\ALL APPLICATION PACKAGES: AccessAllowed (WriteData) <=
: AccessAllowed (WriteData)                                                       <=
```

Reply:
batch, interactive and service shouldn't be allowed.  I'm not sure about the other two.

Reply:
Reading https://techcommunity.microsoft.com/t5/Ask-The-Performance-Team/WS2008-Windows-Service-Hardening/ba-p/372702 Write restricted doesn't give a lot of value here either.

Reply:
We should make sure our MSIX can write without `APPLICATION PACKAGE AUTHORITY\ALL APPLICATION PACKAGES`